lib/rucio/transfertool/fts3.py

```diff
@@ -686,7 +686,7 @@ def _is_recoverable_fts_overwrite_error(cls, request: dict[str, Any], reason: st
         dst_type = file_metadata.get('dst_type', None)
         METRICS.counter('overwrite.check.{rsetype}.{rse}').labels(rse=file_metadata["dst_rse"], rsetype=dst_type).inc()
 
-        if 'Destination file exists and overwrite is not enabled' in (reason or ''):
+        if 'Destination file exists and overwrite is not enabled' in (reason or '') or 'Destination file exists and is on tape' in (reason or ''):
```

Comment:
LGTM; could also be rewritten like this, which might look a bit nicer in case we need to add more messages in the future

```suggestion
        if reason and any(msg in reason for msg in (
        'Destination file exists and overwrite is not enabled',
        'Destination file exists and is on tape'
        )):
```

But it's definitely already clear in your version, so feel free to do either option

Reply:
Thanks Riccardo, yours is cleaner. I've applied it. Will squash the commits now